c10/core/ScalarType.h

```diff
@@ -309,6 +315,8 @@ static inline bool isSignedType(ScalarType t) {
     return std::numeric_limits<ctype>::is_signed;
 
   switch (t) {
+    case ScalarType::Bits16:
+      return false;
```

Comment:
Alternately, could raise an error here, since it's not really defined if bits is signed or not.